kubernetes-templates/weave-cloud/README.md

```diff
@@ -0,0 +1,21 @@
+# Weave Cloud Suite
+
+Weave Cloud provides all you need to implement Continuous Delivery with hosted Prometheus monitoring and visual distributed system analysis/debug tools.
```

Comment:
=> "Weave Cloud is a hosted add-on to Kubernetes which provides Continuous Delivery, Prometheus for Monitoring and a visual dashboard for troubleshooting & debugging microservices"